scraper/src/scraper/providers/anthropic.py

```
@@ -156,37 +160,45 @@ def _build_models_from_results(
     "Claude Sonnet 4.6":        "tag-flag",
     "Claude Haiku 4.5":         "tag-mini",
     "Claude Haiku 3.5":         "tag-mini",
     "Claude Opus 4.1 (Legacy)": "tag-leg",
 }
 _SUB_JA = {
+    "Claude Fable 5":           "最上位モデル / 1M ctx / 長期エージェント最強 / thinking常時ON",
     "Claude Opus 4.8":          "2026年5月 / 1M ctx / Adaptive thinking / 最新フラッグシップ",
     "Claude Opus 4.7":          "SWE-bench 87.6% / コーディング特化 / Apr 2026",
     "Claude Opus 4.6":          "旧フラッグシップ / エージェントチーム / 1M ctx",
-    "Claude Sonnet 4.6":        "バランス最適 / 200K ctx",
+    _CLAUDE_SONNET_5:           "最新Sonnet / 8/31まで促進価格 $2/$10（以降 $3/$15）",
```

Ruff (0.15.21)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)